docs/source/refs/additional_resources.rst

```diff
@@ -0,0 +1,32 @@
+Additional Resources
```

Comment:
Adding a comment here to prevent merging this by accident. This should be merged after Isaac Sim docs become public.